tests/test_scdip.py

```diff
@@ -410,11 +414,57 @@ def test_no_resources_content(self):
         content = null_result_container.find_element_by_css_selector('.col')
         self.assertEqual(content.text, data['content'])
 
+    #Test that a specific resource is found
+    def test_resource_present(self):
+        data = self.run_script(f'tests/scripts/{self.func_name()}.json')
+        self.validate_assertion_data(data, ["category","resource_name", "resource_content"])
+        results = WebDriverWait(self.browser, 10).until(
+            EC.presence_of_element_located((By.CSS_SELECTOR, "#container-results")),
+            "Failed to locate results container"
+        )
+        # look for category
+        cat_rows = results.find_elements_by_css_selector('.row')
+
+        category = next((row for row in cat_rows if row.find_element_by_css_selector('[role=heading]').text == data['category']), None)
+        self.assertIsNotNone(category)
+
+        # look for resource
+        resources = category.find_elements_by_css_selector('[type=resource]')
+        resource = next((res for res in resources if res.find_element_by_css_selector('.headline').text == data['resource_name']), None)
+        self.assertIsNotNone(resource)
+        
+        # check resource content
+        content = resource.find_element_by_css_selector('.v-card__subtitle').text
+        self.assertEqual(content, data['resource_content'])
+
+ 
+    #Test that a specific resource is absent
+    def test_resource_absent(self):
+        data = self.run_script(f'tests/scripts/{self.func_name()}.json')
+        self.validate_assertion_data(data, ["resource_name"])
+        results = WebDriverWait(self.browser, 10).until(
+            EC.presence_of_element_located((By.CSS_SELECTOR, "#container-results")),
```

Comment:
It is not possible to locate #container-results because this is container is not displayed when there are no results. Please can you adapt the test?

Reply:
Incorrect .json steps were being used to run this test